merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/CreateDatasetPartitionsAction.java

```diff
@@ -8,7 +8,7 @@
 
 /*package-local*/ final class CreateDatasetPartitionsAction implements AutoCloseable {
   private static final @Language("SQL") String sql = """
-    select from allocate_dataset_partitions(?);
+    select from allocate_dataset_partitions(?::integer);
```

Comment:
Oh, this is very depressing. Do you know what specific error comes up if you leave the type annotation off?

Reply:
Makes sense. [The `apply` method in this class](https://github.com/NASA-AMMOS/aerie/pull/833/files#diff-86342dd34fb02577133fccc89429aee5b12a9c350b4e5a07a7d7f2c613a9e253L20-R23) takes (took?) a long, not an int.